persistence/src/main/java/com/softserve/borda/repositories/BoardColumnRepository.java

```diff
@@ -2,8 +2,20 @@
 
 import com.softserve.borda.entities.BoardColumn;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
 
 @Repository
 public interface BoardColumnRepository extends JpaRepository<BoardColumn, Long> {
+    List<BoardColumn> getAllBoardColumnsByBoardId(Long boardId);
+
+    @Transactional
```

Comment:
I think, it should be on service layer
More info you can find there - https://habr.com/ru/post/532000/

Reply:
Thanks, fixed. Will read info